Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.cpp

```diff
@@ -1041,6 +1041,19 @@ void fillVideoInfoColorimetryFromColorSpace(GstVideoInfo* info, const PlatformVi
         GST_VIDEO_INFO_COLORIMETRY(info).range = GST_VIDEO_COLOR_RANGE_UNKNOWN;
 }
 
+void configureVideoDecoderForHarnessing(const GRefPtr<GstElement>& element)
+{
+    auto elementHasProperty = [&](const char* name) -> bool {
```

Comment:
It is not the first time I see this, please strongly consider moving this to GStreamerCommon and refactor all appearances. It does not have to be in this PR though.